tests/ports/esp32/ssl_dtls.py

```diff
@@ -0,0 +1,17 @@
+# Very basic test of ssl.SSLContext DTLS.
+
+try:
+    import ssl
+    import ssl.PROTOCOL_DTLS_CLIENT
+    import ssl.PROTOCOL_DTLS_SERVER
+except ImportError:
+    print("SKIP")
+    raise SystemExit
+except NameError:
```

Comment:
Instead of separate except clauses, use `except (ImportError, NameError)` (actually `NameError` might need to be `AttributeError`).

Reply:
Done!